Allow logos for blindfolded solving (#1265)
thewca/wca-regulations: wca-guidelines.md

- 3k2b+) [CLARIFICATION] If a non-permitted puzzle is found during the course of a 3x3x3 Multi-Blind attempt, the puzzle must not be exchanged or removed from the attempt and must be counted as unsolved if the entire attempt is not disqualified.
- 3k2b++) [EXAMPLE] Suppose the competitor attempts 10 puzzles and solves 8 successfully. If it is found that two solved puzzles and one unsolved puzzle are not permitted, the competitor is considered to have a result with 6 solved puzzles out of 10 (rather than DNF), at the discretion of the WCA Delegate.
- 3l+) [ADDITION] Logos may feature any reasonable design that does not give a conspicuous advantage (e.g. encoding information that could be used to cheat), and leave the colored part clearly recognizable. Unconventional logos are only permitted at the discretion of the WCA Delegate.

> - 3l++) [CLARIFICATION] Logos are not permitted for 3x3x3 Multi-Blind.

Author: We probably don't actually need this

> - 3l++) [CLARIFICATION] Logos are not permitted for 3x3x3 Multi-Blind.

Author: ```suggestion
- 3l++) [CLARIFICATION] Logos are permitted for 3x3x3 Blindfolded, but not any other blindfolded events, including 3x3x3 Multi-Blind.
```

@ohexter IMO we either make this change, or remove the guideline entirely

## <article-B><blindfolded><blindfoldedsolving> Article B: Blindfolded Solving

> - B1+) [REMINDER] Other than any permitted logos (see [Regulation 3l](regulations:regulation:3l)), the competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.

Reviewer: I think the first sentence is wordy and somewhat difficult to parse. How about this?

```suggestion
- B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving. Exception: For 3x3x3 Blindfolded and 5x5x5 Blindfolded, the puzzle may have any logos permitted by [Regulation 3l](regulations:regulation:3l).
```

Author: Hmm, I'm not sure if it is really an improvement - it seems a bit weird to give an exception that applies for half of the events

Reviewer: Thanks for the response! I see where you're coming from, but I wouldn't view it that way. The focus of the guideline is the prohibition of the vast majority of "features that distinguish similar pieces", and logos for 3BLD/5BLD are a rare case of such distinguishable features being allowed. So I'd say calling it an exception is appropriate.

Under the originally proposed changes, Article B does not mention that logos are allowed for 3BLD/5BLD but not 4BLD. I would guess that most competitors look to Article B when they want to learn about the blindfolded regulations. But unless they happen to jump around the regulations by clicking 3l and leaving Article B right as they've started reading it, they will come out of it fully unaware that only *some* blindfolded events allow logos. Name-dropping 3BLD and 5BLD makes it clear to such readers that only some blindfolded events allow logos.

As far as the wording goes, I think listing the exception at the end of the guideline makes for a clearer reading experience. The reader doesn't know what "Other than any permitted logos" is meant to communicate until they're halfway through the guideline, and then they'll have to go back and reread the exception at the start of the guideline now that they have the proper context for it. That's a moderately confusing read. On the other hand, if the exception is listed at the end of the guideline, it's immediately obvious which rule it is an exception to, and the use of the word "Exception" primes the reader for an important exception to the guideline they just read.

## <article-B><blindfolded><blindfoldedsolving> Article B: Blindfolded Solving

> - B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving. Exception: For 3x3x3 Blindfolded and 5x5x5 Blindfolded, the puzzle may have any logos permitted by [Regulation 3l](regulations:regulation:3l).

Author: ```suggestion
- B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving. Exception: For 3x3x3 Blindfolded, the puzzle may have any logos permitted by [Regulation 3l](regulations:regulation:3l).
```

- B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must (ask and) receive confirmation from the judge each time before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without confirmation from the judge: disqualification of the attempt (DNF).
- B2f+) [CLARIFICATION] Since the competitor starts the solve by lifting the cover, penalties for starting the timer and for touching the cover while starting the attempt do not apply (see [Regulation B2b](regulations:regulation:B2b) and [Regulation B2c](regulations:regulation:B2c)).
- B4c+) [ADDITION] If the judge forgot to put a sight blocker, or places it in a way that the competitor's view of the puzzle is not fully blocked, the WCA Delegate should grant an extra attempt.
- B4c++) [ADDITION] If the WCA Delegate does not suspect that the competitor intentionally violated any WCA Regulations, the original attempt may stand.

> - B4d+) [CLARIFICATION] Competitors are permitted to feel a permitted logo (see [Regulation 3l](regulations:regulation:3l)) during the blindfolded phase.

Author: ```suggestion
- B4d+) [CLARIFICATION] Competitors are permitted to determine the location of a permitted logo by touch (see [Regulation 3l](regulations:regulation:3l)) during the blindfolded phase. 
```

> - B4d+) [CLARIFICATION] Competitors are permitted to determine the location of a permitted logo by touch (see [Regulation 3l](regulations:regulation:3l)) during the blindfolded phase. 

Reviewer 2: ```suggestion
- B4d+) [CLARIFICATION] For 3x3x3 Blindfolded, competitors are permitted to determine the location of a permitted logo (see [Regulation 3l](regulations:regulation:3l)) by touch during the blindfolded phase. 
```
Redundant, but to avoid any confusion
